spec/lib/feature_management_spec.rb

```diff
@@ -216,13 +216,13 @@
       context 'with identity verification enabled' do
         let(:enable_identity_verification) { true }
 
-        it { expect(no_pii_mode?).to eq(false) }
+        it { expect(fake_banner_mode?).to eq(true) }
       end
 
       context 'with identity verification disabled' do
         let(:enable_identity_verification) { false }
 
-        it { expect(no_pii_mode?).to eq(false) }
+        it { expect(fake_banner_mode?).to eq(true) }
```

Comment:
Can we please update all of these tests to match the new conditions? These old scenarios don't apply anymore.